docs/pages/setup/reference/config.mdx

```diff
@@ -507,6 +507,25 @@ ssh_service:
     # Enables/disables TCP forwarding. Default is 'true'
     port_forwarding: true
 
+    # When enabled, User's with the "permit_x11_forwarding" role option
```

Comment:
```suggestion
    # When x11.enabled is set to yes, users with the "permit_x11_forwarding" role option
```
This is slightly more than 80 chars, so we'll probably need to reflow this comment